Essentials/src/main/java/net/essentialsx/api/v2/events/chat/LocalChatEvent.java

```diff
@@ -0,0 +1,28 @@
+package net.essentialsx.api.v2.events.chat;
+
+import net.essentialsx.api.v2.ChatType;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.HandlerList;
+
+import java.util.Set;
+
+/**
+ * Fired when a player uses local chat
+ */
+public class LocalChatEvent extends ChatEvent implements Cancellable {
```

Comment:
Would like a a `getRadius` method here to see the local chat radius

Reply:
Added in [7bd890d](https://github.com/EssentialsX/Essentials/pull/4683/commits/7bd890db1b02417e72ad6be4266e17255d63b4ed) ([lines 23-32](https://github.com/EssentialsX/Essentials/pull/4683/commits/7bd890db1b02417e72ad6be4266e17255d63b4ed#diff-3882453ce03b1c2e3ac877526b21f1eafdbc5cc3713e6b4ac4dcc65494f477ddR23-R32)).